internal/linker/patches/0001-add-dynamic-magic-value-and-unexported-function-name.patch

```diff
@@ -0,0 +1,83 @@
+From 57e413e5898b093a1461359ffe860545eb08b965 Mon Sep 17 00:00:00 2001
```

Comment:
please keep the patches separate, one on top of the other. that is, we want our branch to have one commit on top of upstream master for each different change that we want to make. I think two patches now makes sense - one for magic numbers, another for stripping func names. we don't want to end up with 50 patches, but we also don't want everything in one massive patch and commit.

Reply:
2 patches for 1 file = 2 git process creation, is this ok? (+ need to write more logic for sequential file patching)

Reply:
git-apply takes a list of patches, as far as I can tell :)

Reply:
git apply can apply multiple patches from stdin?????

Reply:
Ah, not sure. I was only seeing the man page, which accepts multiple path arguments. I guess writing multiple temp files is better than making multiple git calls?

Either way I'm not worried as we'll hopefully remove git from the equation soon.

Reply:
I'm surprised, just pass a 2 patches to stdin and it just works